packages/react/src/patterns/blocks/ContentBlockMedia/ContentBlockMedia.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import ContentBlockMediaGroup from './ContentBlockMediaGroup';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import FeaturedLink from '../FeaturedLink/FeaturedLink';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Use cases pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.border Use cases border
+ * @param {string} props.copy Use cases short copy to support the heading
+ * @param {string} props.theme Use cases color theme
+ * @param {string} props.heading Use cases heading
+ * @param {Array} props.mediaGroup mediaGroup array with heading, image and lists
+ * @param {object} props.featuredLink featuredLink object
+ * @returns {object} JSX Object
+ */
+const ContentBlockMedia = ({
+  border,
+  copy,
+  theme,
+  heading,
+  mediaGroup,
+  featuredLink,
+}) => {
+  const setBorder = _setBorder(border);
+  const setTheme = _setTheme(theme);
+
+  return (
+    <ContentBlock
+      heading={heading}
+      copy={copy}
+      data-autoid={`${stablePrefix}--content-block-media`}
+      customClassName={`${prefix}--content-block-media ${prefix}--col-lg-8 ${setBorder} ${setTheme}`}>
+      {_rendermediaGroup(mediaGroup)}
+      {featuredLink && <FeaturedLink {...featuredLink} />}
```

Comment:
the featured link should actually be part of the ContentBlock CTA, and that is specifically set to only use the feature card

Reply:
Ah, gotcha. I didn't realize there was a new `CTA` component when I started this.